rconweb/api/votemap.py

```diff
@@ -109,7 +115,6 @@ def reset_votemap_state(request):
             command="reset_votemap_state",
         )
     audit("reset_votemap_state", request, {})
-    v = VoteMap()
```

Comment:
You said it wasn't instantiating a VoteMap instance here. Clearly it did, but you must've accidentally removed it. Not a big deal, though preferably you only instantiate it after you check if the request method is indeed POST.

Reply:
Now that I understand better, it was instantiating it below but we were trying to use it in the response for non-POST requests.

I changed it to an error message instead.